internal/gqlparser/validator/imported/readme.md

```diff
@@ -0,0 +1,13 @@
+# graphql.js spec importer
```

Comment:
Is it worth keeping this? I don't really see us regenerate the graphql spec like this...